### ISSUE TITLE
Language support for systemd files

### ISSUE THREAD

**Issue by dwt** [zed-industries/extensions#1715]
### Check for existing issues

- [X] Completed

### Describe the feature

Systemd service, unit, target, timer files all share a specific syntax and would greatly profit from syntax highlighting and language server support.

### Environment

Zed: v0.162.5 (Zed)
OS: macOS 15.1.1
Memory: 36 GiB
Architecture: aarch64

**Comment by ProPrak01** [comment]
Thank you for highlighting the need for systemd file support. I am currently working on an extension to address this. Once I make some progress, I’ll share mockups and updates here.

**Comment by yodatak** [comment]
systemd Language server 
https://github.com/psacawa/systemd-language-server

**Comment by dwt** [comment]
This doesn't seem to be installable via a plugin yet, am I missing something obvious?

**Comment by yodatak** [comment]
yes you right is just easy to develop from existing language server for somebody ready this issue

**Comment by jaredwy** [comment]
the licence choice may be an issue...
